assets/js/components/settings/EntityOwnershipChangeNotice.js

```diff
@@ -30,38 +35,59 @@ import { CORE_USER } from '../../googlesitekit/datastore/user/constants';
 import { useFeature } from '../../hooks/useFeature';
 import SettingsNotice from '../SettingsNotice/SettingsNotice';
 import { TYPE_WARNING } from '../SettingsNotice/utils';
+import { listFormat } from '../../util';
 const { useSelect } = Data;
 
 export default function EntityOwnershipChangeNotice( { slug } ) {
 	const isDashboardSharingEnabled = useFeature( 'dashboardSharing' );
 
-	const storeName = useSelect( ( select ) =>
-		select( CORE_MODULES ).getModuleStoreName( slug )
-	);
+	const slugs = Array.isArray( slug ) ? slug : [ slug ];
 
-	const haveOwnedSettingsChanged = useSelect( ( select ) =>
-		select( storeName )?.haveOwnedSettingsChanged()
-	);
+	const storeNames = useSelect( ( select ) => {
+		const { getModuleStoreName } = select( CORE_MODULES );
 
-	const moduleOwnerID = useSelect( ( select ) =>
-		select( storeName )?.getOwnerID()
-	);
+		return slugs.reduce( ( acc, currentSlug ) => {
+			const storeName = getModuleStoreName( currentSlug );
+			if ( storeName ) {
+				acc[ currentSlug ] = storeName;
+			}
+			return acc;
+		}, {} );
+	} );
 
-	const loggedInUserID = useSelect( ( select ) =>
-		select( CORE_USER ).getID()
+	const haveOwnedSettingsChanged = useSelect( ( select ) =>
+		Object.keys( storeNames ).reduce( ( acc, currentSlug ) => {
+			const storeName = storeNames[ currentSlug ];
+			const moduleOwnerID = select( storeName )?.getOwnerID();
+			const loggedInUserID = select( CORE_USER ).getID();
+			const haveSettingsChanged =
+				select( storeName )?.haveOwnedSettingsChanged();
+			if ( haveSettingsChanged && moduleOwnerID === loggedInUserID ) {
```

Comment:
This operator is wrong - we should only be showing the notice if the module owner _is not_ the logged in user.

Note how previously we were returning `null` if `moduleOwnerID === loggedInUserID`.

```suggestion
			if ( haveSettingsChanged && moduleOwnerID !== loggedInUserID ) {
```